CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## TO BE RELEASED
 
+* recipes to install and configure the utility node to host the capture-agent-manager
+  flask-gunicorn webapp
+
+* *REQUIRES EDITS TO THE CLUSTER CONFIG* *REQUIRES MANUAL CHEF RECIPE RUNS*:
```

Comment:
I would update this language to make sure it's understood this is optional and only for the DCE production cluster.